src/components/CodeView/styles.module.scss

```diff
@@ -0,0 +1,18 @@
+@import '../../scss/variables';
+
+.CodeView {
+  border: 1px solid $light-gray;
+  padding: 10px;
+}
+
+.lineNumbers {
+  display: grid;
+  float: left;
+  padding-right: 10px;
+
+  * {
```

Comment:
Why not just put a class on all the spans like `.lineNumber` and select that?

Reply:
:+1: